pkgdb2/templates/packager.html

```diff
@@ -11,9 +11,9 @@
   {{ packager | avatar | safe }}
   {{ packager }}</h1> (<a class="fas" target="_blank"
     {% if packager.startswith('group::') %}
-        href="https://admin.fedoraproject.org/accounts/group/view/{{ packager.replace('group::', '') }}"
+        href="{{ config['PKGDB2_FAS_URL'] }}/group/view/{{ packager.replace('group::', '') }}"
```

Comment:
It's None right now but pkgdb will have more problem if it isn't set (like it won't be able to check if someone is a packager) so I am pretty confident that this will always be set.